frontend/installation-ui/web/micro-ui-internals/packages/modules/qc/src/components/FacilityDetails/SystemParameterReport.js

```
@@ -35,6 +43,45 @@ const SystemParameterReport = ({ file }) => {
+            style={{
+              border: "1px solid #eee",
+              borderRadius: "6px",
+              padding: "16px",
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "space-between",
+              minWidth: "fit-content",
+              width: "20%",
+              position: "relative"
+            }}
+          >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)